### PR TITLE
feat(store): add Megekko

### PR DESCRIPTION
**Pull request by Hangur**
feat: Megekko (NL) added to the existing stores.

### Description

- Added 1 new store: Megekko (NL). Currently only products listed are from the RTX 3080 Series.
- Also added a few new models for RTX 3080.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -272,13 +273,13 @@ environment variables are **optional**._
 | `amd` | `5600x`, `5800x`, `5900x`, `5950x`, `amd reference` |
 | `asus` | `dual`, `dual oc`, `strix`, `strix oc`, `tuf`, `tuf oc` |
 | `corsair` | `750 platinum`, `600 platinum` |
-| `evga` | `ftw3`, `ftw3 ultra`, `xc3`, `xc3 black`, `xc3 ultra` |
+| `evga` | `ftw3`, `ftw3 ultra`, `xc3`, `xc3 black`, `x3 gaming`, `xc3 ultra` |
 | `gainward` | `phantom gs`, `phoenix`, `phoenix gs`, `phoenix gs oc` |
-| `gigabyte` | `aorus master`, `aorus xtreme`, `eagle`, `eagle oc`, `gaming`, `gaming oc`, `turbo`, `vision`, `vision oc` |
-| `inno3d` | `gaming x3`, `ichill x3`, `ichill x4`, `twin x2 oc` |
+| `gigabyte` | `aorus master`, `aorus xtreme`, `aurus xtreme waterforce`, `eagle`, `eagle oc`, `gaming`, `gaming oc`, `turbo`, `vision`, `vision oc` |
```
Comment (FatalErrorXDevs) [inline README.md:278]: should that not be aorus?

File: src/store/model/megekko.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+			container: '.product_detail .text_red',
+			text: ['minimaal 10 dagen']
+		}
+	},
+	links: [
+		{
+			brand: 'test:brand',
+			model: 'test:model',
+			series: 'test:series',
+			url: 'https://www.coolblue.nl/product/826844/'
```
Comment (FatalErrorXDevs) [inline src/store/model/megekko.ts:23]: test link is still for cool blue, please use something from megekko

**Review by FatalErrorXDevs — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Hangur) [reply README.md:278]: thanks for catching, will change :+1:

File: src/store/model/megekko.ts (hunk 2 above)
Comment (Hangur) [reply src/store/model/megekko.ts:23]: ugh... my bad, will fix right now.

**Review by jef — CHANGES_REQUESTED** [review]
Looks pretty good overall!

Just a few things:

- Can you remove `x3 gaming` in favor of `x3`. These are all ["gaming" cards](https://www.evga.com/products/productlist.aspx?type=0&family=GeForce+30+Series+Family&chipset=RTX+3080).
- Can you merge from `main` and update the docs in their new location?

Thank you!

**Comment by jef** [comment]
Also, it seems to me that _all_ these cards are in stock? Did I just happen to check at the right time? Or is this accurate information?

**Comment by Hangur** [comment]
> Looks pretty good overall!
> 
> Just a few things:
> 
> * Can you remove `x3 gaming` in favor of `x3`. These are all ["gaming" cards](https://www.evga.com/products/productlist.aspx?type=0&family=GeForce+30+Series+Family&chipset=RTX+3080).
> * Can you merge from `main` and update the docs in their new location?
> 
> Thank you!

Will do, thanks! :+1: 

> Also, it seems to me that _all_ these cards are in stock? Did I just happen to check at the right time? Or is this accurate information?

I am not sure, just checked the site: 

megekko 

and for all 3080's it says:

```Levertijd: minimaal 10 dagen```

which in Dutch means that delivery will take at least 10 days. But in **practice it means pre-order** and most likely the delivery is in 2021.

**Comment by FatalErrorXDevs** [comment]
> Also, it seems to me that _all_ these cards are in stock? Did I just happen to check at the right time? Or is this accurate information?

He is right about the minimaal 10 dagen message, this store tends to let you order, regardless of if they can deliver. In the past i ordered a 3900X from AMD at launch and it took them 3 months to deliver, the whole time they had that message up.

**Comment by Hangur** [comment]
> He is right about the minimaal 10 dagen message, this store tends to let you order, regardless of if they can deliver. In the past i ordered a 3900X from AMD at launch and it took them 3 months to deliver, the whole time they had that message up.

(s)he ;)

**Comment by jef** [comment]
> which in Dutch means that delivery will take at least 10 days. But in practice it means pre-order and most likely the delivery is in 2021.

Interesting... Okay! Well in that case. I think we're good to go!

Thank you very much for your contribution!

**Review by jef — APPROVED** [review]

**Pull request merged**